x-pack/legacy/plugins/canvas/canvas_plugin_src/strings/functions/pie.ts

```diff
@@ -11,47 +11,57 @@ import { FunctionFactory, Position } from '../../functions/types';
 
 export const help: FunctionHelp<FunctionFactory<typeof pie>> = {
   help: i18n.translate('xpack.canvas.functions.pieHelpText', {
-    defaultMessage: 'Configure a pie chart element',
+    defaultMessage: 'Configure a pie chart element.',
   }),
   args: {
-    palette: i18n.translate('xpack.canvas.functions.pie.args.paletteHelpText', {
-      defaultMessage: 'A {palette} object for describing the colors to use on this pie',
-      values: {
-        palette: 'palette',
-      },
-    }),
-    seriesStyle: i18n.translate('xpack.canvas.functions.pie.args.seriesStyleHelpText', {
-      defaultMessage: 'A style of a specific series',
-    }),
-    radius: i18n.translate('xpack.canvas.functions.pie.args.radiusHelpText', {
+    font: i18n.translate('xpack.canvas.functions.pie.args.fontHelpText', {
       defaultMessage:
-        'Radius of the pie as a percentage (between 0 and 1) of the available ' +
-        'space. Set to `{auto}` to automatically set radius',
+        'The {css} font properties for the labels. For example, {fontFamily} or {fontWeight}.',
       values: {
-        auto: 'auto',
+        css: 'CSS',
+        fontFamily: 'font-family',
+        fontWeight: 'font-weight',
       },
     }),
     hole: i18n.translate('xpack.canvas.functions.pie.args.holeHelpText', {
-      defaultMessage: 'Draw a hole in the pie, 0-100, as a percentage of the pie radius',
-    }),
-    labels: i18n.translate('xpack.canvas.functions.pie.args.labelsHelpText', {
-      defaultMessage: 'Show pie labels?',
+      defaultMessage: 'Draws a hole in the pie, 0-100, as a percentage of the pie radius.',
     }),
     labelRadius: i18n.translate('xpack.canvas.functions.pie.args.labelRadiusHelpText', {
-      defaultMessage: 'Percentage of area of container to use as radius for the label circle',
+      defaultMessage:
+        'The percentage of the container area to use as a radius for the label circle.',
     }),
-    font: i18n.translate('xpack.canvas.functions.pie.args.fontHelpText', {
-      defaultMessage: 'Label font',
+    labels: i18n.translate('xpack.canvas.functions.pie.args.labelsHelpText', {
+      defaultMessage: 'Displays the pie labels?',
```

Comment:
This seems like awkward phrasing... almost as if Canvas is wondering if it will work?  Consider "Display the pie labels?"...